packages/loader/container-loader/src/loader.ts

```diff
@@ -195,6 +196,7 @@ export class Loader extends EventEmitter implements ILoader {
             },
             this.documentServiceFactory,
             this.resolver,
+            this.containers,
```

Comment:
I'm not sure I like that. Container has access to loader, so I'd rather go other way around - Container calling into loader to remember itself. That way we can have single logic about what is cachable, as not every container is cachable (based on request.header flags) - loader may still chose not to cache it

Reply:
Added api to cache container.